src/test/java/joelrojastest/Requests.java

```diff
@@ -0,0 +1,39 @@
+package joelrojastest;
+
+import entities.Project;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+
+import static io.restassured.RestAssured.given;
+
+public class Requests {
+    public static final String TOKEN = "acea0f034eb75c9d8ed7ca11d0f74376f5b7bba8";
```

Comment:
move this token and url to config file

Reply:
I created a configuration file and move them to it.